### DISCUSSION TITLE
Rad Pro: Dimmed display sleep

### DISCUSSION THREAD

**Discussion by niloc-ray** [Gissio/radpro#119] [Ideas]
Hi have the FNIRST GC-01 and I love this firmware. One thing that is missing compared to the stock firmware is, on the stock firmware the display sleep only turned off the backlight, not the whole display so it can still be read. This would be nice to have as an option.

**Comment by gc01noob** [comment]
try the ~Theme~ Brightness section of the menu, I like the Mid setting. You can also expand the timeout time - not quite what you mentioned but close.
